projects/js-packages/components/components/status/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { __ } from '@wordpress/i18n';
+import classNames from 'classnames';
+import styles from './style.module.scss';
+
+interface StatusProps {
+	status?: 'active' | 'error' | 'inactive' | 'action' | 'initializing';
+	label?: string;
+	className?: string;
+}
+
+const Status = ( { className, label, status = 'inactive', ...rest }: StatusProps ): JSX.Element => {
+	const defaultLabels: Record< string, string > = {
+		active: __( 'Active', 'jetpack' ),
+		error: __( 'Error', 'jetpack' ),
+		action: __( 'Action needed', 'jetpack' ),
+		inactive: __( 'Inactive', 'jetpack' ),
+		initializing: __( 'Setting up', 'jetpack' ),
+	};
+
+	return (
+		<span
+			className={ classNames(
+				styles.status,
+				{
+					[ styles[ `status--${ status }` ] ]: status,
+				},
+				className
+			) }
+			{ ...rest }
```

Comment:
"What is the purpose of propagating the rest of the props in this context? Can you identify any unforeseen props that might be applied to the span element?"

Reply:
Removed: https://github.com/Automattic/jetpack/pull/30209/commits/097cca19811ecab4023604347f2f7136930bf731

I had included the prop spreading just to make it flexible, i.e. add event handlers or extra HTML properties as needed during implementation. But I see how this would make the component more complex/dangerous to work with, so I am happy to have removed it :+1: